package.json

```diff
@@ -7,6 +7,8 @@
     "prepare": "is-ci || husky install",
     "template:dev": "TUTORIALKIT_DEV=true pnpm run build && pnpm run --filter=tutorialkit-starter dev",
     "template:build": "pnpm run build && pnpm run --filter=tutorialkit-starter build",
+    "tkit": "pnpm run --filter=tutorialkit.dev dev",
```

Comment:
What if this was `"docs"` instead of `"tkit"`?

Reply:
So first I had `docs`. It had 2 minor "issues"

1. You can't run `pnpm docs`, because it opens the pnpm documentation. So you need to run `pnpm run docs`.
2. If we are moving the demo over to this repo as well, it might be confusing?

I don't know, it's not big of a deal and we could always just have `pnpm run demo` and `pnpm run docs`. Maybe that's better?

Reply:
Oh I quite like `pnpm run demo` and `pnpm run docs`! :star2: 

It's a bummer that `pnpm docs` doesn't work. Maybe we could have `docs-site` instead? 

Although I think I prefer `pnpm run docs`.

Reply:
Yeah I agree. I'll update it.